src/components/dashboard/DashboardComponent.tsx

```diff
@@ -1,10 +1,73 @@
+"use client";
+
+import { BagIcon } from "@/icons";
+import CardContainer from "./components/CardContainer";
+import { FlameIcon } from "lucide-react";
+import { AboutStore } from "./components/AboutStore";
+import { TopProducts } from "./components/TopProducts";
+import { Statistics } from "./components/Statistics";
+import { SellsPerMoth } from "./components/SellsPerMoth";
 
 const DashboardComponent = () => {
-    return (
-        <>
-            <h1 className="text-white text-2xl font-bold">Dashboard</h1>
-        </>
-    )
-}
-
-export default DashboardComponent
\ No newline at end of file
+  return (
+    <div className="min-h-screen space-y-24 bg-[#1a1a2e] text-white p-4 sm:p-6 lg:p-8">
```

Comment:
```suggestion
    <div className="min-h-screen space-y-24 text-white p-4 sm:p-6 lg:p-8">
```
please, delete the bg-color

Reply:
Done